models/topics.models.js

```diff
@@ -9,4 +9,12 @@ exports.fetchTopics=()=>{
 
 exports.invalidPaths = () => {
     return Promise.reject({ status: 404, msg: "Not Found" });
-};
\ No newline at end of file
+};
+
+exports.selectEndpoints = () => {
+    return fs.readFile(`${process.cwd()}/endpoints.json`,'utf-8')
```

Comment:
Dont forget just like how you did in your test you can directly require in .json files so you dont need to use the fs library.